### ISSUE TITLE
CommandExecutors for multiple types

### ISSUE THREAD

**Issue by MatsG23** [JorelAli/CommandAPI#231]
I suggest a `CommandExecutor` where you can specify multiple types of `CommandSender`s that will run the same `execute` function.

_Example:_
```kotlin
CommandAPICommand("somerandomcommand")
.executes(CommandExecutor {sender, args -> 
     sender.sendMessage("Command ran successfully!")
}, PlayerCommandExecutor::class.java, ConsoleCommandExecutor::class.java)
```

**Comment by JorelAli** [comment]
I like this suggestion, it sounds very useful!

**Comment by JorelAli** [comment]
Implemented in 7.0.0.
